shared/js/project.js

```diff
@@ -464,6 +496,13 @@ function executeCallback(callback, data) {
 }
 
 /**
+ * is the current project (if any) editable by the logged in user (if any)?
+ */
+function isEditable() {
```

Comment:
Ah, I totally missed that.  Your call: I think `frozen` as a state is probably the right way to do it, and it's easier to enforce on the API side (vs. parsing the `value` hash again).  But it's also more work, and if consumers aren't respecting `hidden` we have other problems (like someone could delete their Big Game project and be in a world of hurt).

I think you're right that ideally `hidden` and `frozen` would both be states.